src/meta/meta_ckpt.c

```diff
@@ -1265,6 +1266,141 @@ __wt_meta_ckptlist_update_config(
     return (ret);
 }
 
+/*
+ * ckpt_blkmod_to_item --
+ *     Extracts the blkmod info in a checkpoint into a bitmap in a WT_ITEM.
+ */
+static int
+ckpt_blkmod_to_item(WT_SESSION_IMPL *session, WT_CKPT *ckpt, const char *id, WT_ITEM *output_item)
```

Comment:
I wonder if these functions, plus the new changes to `__wt_meta_ckptlist_set` might better belong in `cur_backup_incr.c`?  Just because they have similarities.  Or maybe a new file `meta_backup_incr.c`?   I'm not committed to any one of these, just throwing out for consideration.

Reply:
Hi @ddanderson , that's definitely worth considering. I put `ckpt_blkmod_to_item` in `meta_ckpt.c` because the only place it is currently used is in `__wt_meta_ckptlist_set`, but if there's value in moving it, we can do that.